pyanaconda/rescue.py

```diff
@@ -248,6 +246,11 @@ def _unlock_devices(self):
                                               parents=[device],
                                               exists=True)
                         self.storage.devicetree._add_device(luks_dev)
+
+                        # Wait for the device.
+                        # Otherwise, we could get a message about no Linux partitions.
+                        time.sleep(2)
```

Comment:
I have tried udev settle, but that does not solve the problem. I was looking for inspiration in blivet and other places, but couldn't find anything. However, maybe I have overlooked something.